src/python/WMComponent/AgentStatusWatcher/AgentStatusPoller.py

```diff
@@ -255,29 +257,44 @@ def collectWMBSInfo(self):
 
         return results
 
-    def checkProxyLifetime(self, agInfo):
+    def checkCredLifetime(self, agInfo, cred_type):
```

Comment:
Can you rename it to camel case please (credType)?

Reply:
Done